ckanext/oauth2/oauth2.py

```diff
@@ -38,23 +37,24 @@
 
 import jwt
 
-import constants
+from .constants import *
```

Comment:
Wildcard import are a bit dangerous and potentially confusing, but they are not a big issue.